websub-native/src/main/java/io/ballerina/stdlib/websub/NativeStringUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.ballerina.stdlib.websub;
+
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.values.BString;
+
+import java.util.Random;
+
+/**
+ * {@code NativeStringUtils} comprise with methods related to {@code String} manipulation.
+ */
+public class NativeStringUtils {
+    public static BString generateRandomString(int targetStringLength) {
```

Comment:
Can't do in Ballerina ? :)

Reply:
* Refactored the random-string generation logic and moved string-generation to ballerina.